atom/app/atom_main.cc

```diff
@@ -43,6 +43,7 @@ namespace {
 const char kRunAsNode[] = "ELECTRON_RUN_AS_NODE";
 #endif
 
+#if defined(OS_WIN) || BUILDFLAG(ENABLE_RUN_AS_NODE)
 bool IsEnvSet(const char* name) {
```

Comment:
Probably using `ALLOW_UNUSED_TYPE` would be a better option here ?

Reply:
Let's try it out. Thanks!